service/informer/informer.go

```diff
@@ -0,0 +1,216 @@
+package informer
```

Comment:
The informer service added here is the actual coordinator of the business logic. The PR is already awful so I would do the testing in the next PR. Here we got some abstraction sorted and moved code around. The informer is booted during daemon boot in the root service package.